clients/unshaded/src/main/java/tachyon/hadoop/HdfsFileInputStream.java

```diff
@@ -73,7 +77,7 @@
    * @param tachyonConf Tachyon configuration
    * @throws IOException if the underlying file does not exist or its stream cannot be created
    */
-  public HdfsFileInputStream(TachyonFS tfs, long fileId, Path hdfsPath, Configuration conf,
+  public HdfsFileInputStream(TachyonFileSystem tfs, long fileId, Path hdfsPath, Configuration conf,
```

Comment:
Users shouldn't directly use `HdfsFileInputStream`, it is wrapped in Hadoop's `FSDataInputStream`